src/openzaak/conf/includes/base.py

```diff
@@ -591,6 +591,15 @@
     os.path.join(DJANGO_PROJECT_DIR, "notifications", "tests", "files"),
 ]
 
+#
+# VNG-API-COMMON
+#
+NOTIFICATIONS_DISABLED = config(
+    "NOTIFICATIONS_DISABLED",
+    default=False,
+    cast=lambda val: val in ["True", "true", "yes", "1"],
```

Comment:
this is not needed, `config` already takes care of that because the `type` of `default` is `bool` :)

Reply:
Oh, neat :)